.github/workflows/release.yml

```diff
@@ -0,0 +1,102 @@
+---
+
+name: Release
+
+on:
+  pull_request:
+    branches: main
+    types: closed
```

Comment:
This seems wrong. We shouldn't be releasing when ever there is a PR closed on main. For this, I think it should be when ever a new release is created in GitHub. That way we can use the release tag for the flatpak branch, and docker tag.

Reply:
It's structured the same way as the release action for our other repos because the intention is that the release action should create a release tag. That's not included in this branch, but should be the eventual behavior

Reply:
Then we should also include this line: https://github.com/elementary/calculator/blob/master/.github/workflows/release.yml#L15

Reply:
Oops, nice catch!

Reply:
We also need it on the Docker step :+1: